src/image/image.css

```diff
@@ -490,6 +490,12 @@ button.open-inspector {
 .img.loom {
   -webkit-mask-image: url(/images/Loom.svg);
 }
+.img.k-icon {
+  -webkit-mask-image: url(/images/k-icon.svg);
+}
+.img.cmd-icon {
+  -webkit-mask-image: url(/images/cmd-icon.svg);
+}
```

Comment:
This is the original way that we used to import images to devtools. Probably should based our new Icon implementation off of it.